redhat-developer/devspaces
# fix: update python and jupyter to latest stable releases 2023.6.1 / 2023.3.100 (CRW-433, CRW-4111)

### What does this PR do?

fix: update python and jupyter to latest stable releases 2023.6.1 / 2023.3.100 (CRW-433, CRW-4111)

Change-Id: I077a8f6b83557fd8cc3aeeb853e0034d91b4d63c
Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?

https://issues.redhat.com/browse/CRW-4334

### How to test this PR?

See https://issues.redhat.com/browse/CRW-4334?focusedId=22185682&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22185682


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
